src/styles/pagination.scss

```diff
@@ -0,0 +1,54 @@
+.pagination {
+  .pagination-list {
+    background-color: rgba(0, 0, 0, .075);
+  }
+  margin-bottom: 3rem!important;
+  .active {
+    a {
+      color: $white;
+      background-color: $colorpurple;
+      border-radius: 50%;
+      padding: 15px 20px;
+    }
+  }
+  li {
+    padding: 15px 0px 15px 0px;
+    a {
+      color: #495057;
+      padding: 15px 20px;
+      font-weight: bold;
+      outline: none;
+      text-decoration: none
```

Comment:
**Nitpick**

- Consider ending each CSS statement with a semi-colon(;). This way, when the file gets minified, there wouldn't be any issues.